lms/djangoapps/badges/models.py

```diff
@@ -236,7 +236,7 @@ class CourseEventBadgesConfiguration(ConfigurationModel):
         blank=True, default='',
         help_text=_(
             u"Each line is a comma-separated list. The first item in each line is the slug of a badge class to award, "
-            u"with an issuing component of 'edx__course'. The remaining items in each line are the course keys the "
+            u"with an issuing component of 'openedx__course'. The remaining items in each line are the course keys the "
```

Comment:
Make wording same as lines 223 and 231, "The first itme in each line is the slug of a badge class you have created that has the issuing component of..."